app/javascript/controllers/parcours_documents_controller.js

```diff
@@ -1,9 +1,13 @@
 import { Controller } from "@hotwired/stimulus";
 
 export default class extends Controller {
-  enableSubmit(event) {
+  fetchFile() {
+    this.element.querySelector("input[type=file]").click()
```

Comment:
comment ça se fait qu'il arrive à récupérer l'input alors que le bouton sur lequel cette méthode est appelé n'est pas parent de l'input ?

Reply:
Parce que this.element c'est le form auquel est rattaché le controller